### ISSUE TITLE
Question on the checkpoint for Stylegan2 on FQ

### ISSUE THREAD

**Issue by lizaitang** [POSTECH-CVLab/PyTorch-StudioGAN#167]
Excuse me, it seems that the google drive does not contain the checkpoint for StyleGAN on FQ dataset? How can I find it.

**Comment by Hzzone** [comment]
Same problem here. I remember that I have seen the pre-trained models on FFHQ  a few weeks ago.

**Comment by asagar60** [comment]
Yes, i can see links to features and moments but not checkpoint

**Comment by Hzzone** [comment]
@MINGUKKANG Thanks very much for your kindful share!

**Comment by MINGUKKANG** [comment]
Hi,

Here I share the pretrained StyleGAN2 [[weight]](https://drive.google.com/drive/folders/1vbOqcXwb_NGN676-8RjdaPHqj1dSIApR?usp=sharing) on FFHQ 1024x1024.

Thank you.

**Comment by lizaitang** [comment]
Thank you so much! By the way, can we achieve conditional GAN on FFHQ dataset?

**Comment by Hzzone** [comment]
> Thank you so much! By the way, can we achieve conditional GAN on FFHQ dataset?

No, there are no official human labels for FFHQ dataset, but you can do this by borrowing the labels from other research if available.

**Comment by lizaitang** [comment]
Thanks, to achieve this do I need to retrain with the labels?

Hzzone ***@***.***>于2022年9月30日 周五11:50写道：

**Comment by MINGUKKANG** [comment]
We provide all checkpoints we used [here](https://huggingface.co/Mingguksky/PyTorch-StudioGAN/tree/main).

Thank you for your patient.
